### PR TITLE
Formatting docs - backticks and lowercase for 'conda-forge'

### PR DESCRIPTION
**Pull request by chicken-biryani**
This PR aims at fixing #1370

This PR tries to fix the inconsistency across docs by changing mentions of  Conda-forge or Conda-Forge to ``conda-forge`` (Double backticks)

PR Checklist:

- [X] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [X] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] put any other relevant information below

### CODE REVIEW

**Comment by chicken-biryani** [comment]
Hi @beckermr and @kathatherine can you please take a look and let me know if I could better the PR?

(Also, thank god for VS Code 😅🤞)

**Comment by beckermr** [comment]
Looks great!

**Comment by jaimergp** [comment]
I wonder if at this point we should consider RST [substitutions](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/reuse.html#use-a-substitution). That way we always use `|conda-forge|` and standardize in one place how this is rendered. But if this complicates the discussion, we can also leave it aside.

**Comment by chicken-biryani** [comment]
@jaimergp that's okay. If it will help in the wrong run, I'll change it.

Maybe I can experiment with a few lines first. If it doesn't work out I'll drop the commit.

**Comment by hmaarrfk** [comment]
@jaimergp i find that the substitutions increase indirection and make it really difficult to read the source text.

I would personally prefer adding a linting step to catch poorly formatted words in the RST.

**Comment by jaimergp** [comment]
One problem I see with always using `conda-forge` instead of conda-forge is that in RST is not possible to have monospace words in a hyperlink, which would lead to inconsistencies. As a result I am inclined to say we should just have always lowercased, unless it starts a sentence, much like what `conda` suggests in their [CEP README](https://github.com/conda-incubator/ceps#conda-capitalization-standards).

File: src/maintainer/infrastructure.rst [hunk 1]
```diff
@@ -343,7 +343,7 @@ works as follows.
    and some information about the package it is trying to upload.
 3. The webservices server validates the secret token, the integrity of the package, and
    that the package is allowed for the given feedstock.
-4. If all of the validation passes, the package is then copied to the ``conda-forge``
+4. If all of the validation passes, the package is then copied to the conda-forge
```
Comment (jaimergp) [inline src/maintainer/infrastructure.rst:346]:
```suggestion
4. If all of the validation passes, the package is then copied to the ``conda-forge``
```

**Review by kathatherine — APPROVED** [review]
The limitations of rst links make this the best solution. I agree, @jaimergp, and it works for me.

**Pull request merged**